src/Sparrow/Binary/Bits.cs

```diff
@@ -287,7 +287,17 @@ public static int PowerOf2(int v)
 #else
             if (v < powerOf2Table.Length)
                 return powerOf2Table[v];
-            return PowerOf2Internal(v);
+
+            v--;
+            v |= v >> 1;
+            v |= v >> 2;
+            v |= v >> 4;
+            v |= v >> 8;
+            v |= v >> 16;
+            v |= v >> 32;
```

Comment:
This line doesn't make sense? It's on an int?

Reply:
That basically zeroes it,no?

Reply:
True, we can just remove those for `int` ... we were doing it though